python/tidy/servo_tidy/tidy.py

```diff
@@ -17,12 +17,9 @@
 import StringIO
 import subprocess
 import sys
-from licenseck import licenses, licenses_toml, licenses_dep_toml
+import licenseck as licenses
```

Comment:
Nit: This should be `from licenseck import MPL, ...` (we don't wanna prefer `*` even though we import all the names)